bokeh/charts/_data_adapter.py

```diff
@@ -137,6 +138,13 @@ def validate_values(values):
 
             return values
 
+        elif hasattr(values, '__array__'):
```

Comment:
Would something as silly as the following suffice rather than using `blaze.into`?

```python
elif hasattr(values, '__array__'):
    values = pd.DataFrame(np.array(values))
```